en/tools/config-catalog-generator/node_modules/.package-lock.json

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
Is it correct that this file has been added?

Reply:
This file is required to generate the config catalog. Contains updated info about the node modules.